src/theming.jl

```diff
@@ -1,3 +1,53 @@
+if Sys.iswindows()
```

Comment:
```suggestion
# This is a platform-specific way of getting the resolution
# of the current monitor, since each platform offers its own API.
if Sys.iswindows()
```